test/integration/stepper/verticalStepper.spec.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-env mocha */
+import React from 'react';
+import {mount} from 'enzyme';
+import {assert} from 'chai';
+import getMuiTheme from 'src/styles/getMuiTheme';
+import VerticalLinearStepper from '../fixtures/VerticalLinearStepper';
```

Comment:
You raised some good points. That's going to be more code to maintaine but that seems to worth it.